### PR TITLE
Changes per JSON field type support in sse_core

### PR DESCRIPTION
**Pull request by jacobythwaites**
Because the JSON field type is now supported in log consumes, there's no need for a separate `sparkl event` command to extract what was previously json string text and format it.

### CODE REVIEW

File: sparkl_cli/test/test_call.py [hunk 1]
```diff
@@ -84,3 +85,17 @@ def test_call_binary_file(self):
...
+            read=[
+                ("field2", "sparkl_cli/test/data/field2.png")])
+
+        result = sparkl(
+            "call",
+            "Scratch/TestCall/Test4")
+
+        value = auto.cook(result)
+
+        assert(True == value)
```
Comment (reube) [inline sparkl_cli/test/test_call.py]: should be:
`assert(value is True)`

File: sparkl_cli/cmd_listen.py [hunk 2]
```diff
@@ -79,11 +80,19 @@ def on_message(args, term):
...
     the Log operation) and a dict of term fields keyed by name.
 
-    If the -c or -f options are selected, filters the output appropriately.
+    The output is either dumped using the standard struct line format,
+    or as raw JSON, or as cooked JSON.
     """
-    cooked_term = auto.cook(args, term)
-    string_term = json.dumps(cooked_term)
-    print(string_term)
+    if (args.cook):
```
Comment (reube) [inline sparkl_cli/cmd_listen.py]: should be no parantheses

File: sparkl_cli/cmd_listen.py [hunk 3]
```diff
@@ -79,11 +80,19 @@ def on_message(args, term):
...
+        cooked_term = auto.cook(args, term)
+        string_term = json.dumps(cooked_term)
+        print(string_term)
+    elif (args.raw):
```
Comment (reube) [inline sparkl_cli/cmd_listen.py]: ditto

File: sparkl_cli/cookers/auto.py [hunk 4]
```diff
@@ -21,7 +21,10 @@ def cook(args, term):
     2. Invokes the tag-specific cooker, if any.
     """
     if isinstance(term, list):
-        return [cook(args, item) for item in term]
+        if len(term) == 1:
+            return cook(args, term[0])
+        else:
```
Comment (reube) [inline sparkl_cli/cookers/auto.py]: linter complains about the else here (superfluous)

Comment (reube) [reply sparkl_cli/cookers/auto.py]: linter also complains about unused import: CliException

File: sparkl_cli/cmd_listen.py (hunk 2 above)
Comment (jacobythwaites) [reply sparkl_cli/cmd_listen.py]: Fixed

File: sparkl_cli/cmd_listen.py (hunk 3 above)
Comment (jacobythwaites) [reply sparkl_cli/cmd_listen.py]: Fixed.

File: sparkl_cli/cookers/auto.py (hunk 4 above)
Comment (jacobythwaites) [reply sparkl_cli/cookers/auto.py]: Fixed.

**Pull request merged**